enterprise/backend/test/metabase_enterprise/sandbox/pulse_test.clj

```diff
@@ -5,8 +5,8 @@
             [medley.core :as m]
             [metabase.email.messages :as messages]
             [metabase.models :refer [Card Pulse PulseCard PulseChannel PulseChannelRecipient]]
-            [metabase.models.pulse :as models.pulse]
-            [metabase.pulse :as pulse]
+            [metabase.models.pulse :as pulse]
+            metabase.pulse
```

Comment:
This is an error here i believe. We have `pulse/send-pulse!` in this namespace from `metabase.pulse/send-pulse!`. But this switches it to `metabase.models.pulse/send-pulse!` which does not exist.

Reply:
oh i see you used the fully qualified name below. I actually really like that.